system/common.xml

```diff
@@ -337,7 +343,7 @@
 				</then>
 		</if>
 		<copy todir="${SYSTEMTEST_DEST}/systemtest_prereqs/">
-			<fileset dir="${TEST_ROOT}/systemtest_prereqs/" includes="**" />
+			<fileset dir="${system_lib_dir}" includes="**"/>
```

Comment:
Why do we still need to copy to `systemtest_prereqs` dir?

Reply:
No, we don't need to copy to the `systemtest_prereqs` directory. I had kept it to check if local tests might need it; apparently, it's not needed. I will remove it.